.github/workflows/deno_test.yml

```diff
@@ -28,37 +28,62 @@ jobs:
         with:
           deno-version: v1.x
       # Checkout target branch and run tests
-      - name: Run Tests with Coverage on target branch
-        run: KAKAROT_ADDRESS=0x1 deno test --allow-env --coverage=cov_profile
-      - name: Generate Coverage Report on target branch
-        run: deno coverage cov_profile --html
-      - name: Install bc
+      - name: Install lcov
         run: |
           sudo apt-get update
-          sudo apt-get install -y bc
+          sudo apt-get install -y lcov
+      - name: Run Tests with Coverage on target branch
+        run: |
+          KAKAROT_ADDRESS=0x1 deno test --allow-env --coverage=cov_profile_1 --ignore=indexer/src/transform.test.ts,indexer/src/testnetData.test.ts
+          deno coverage cov_profile_1 --lcov > cov_profile_1.lcov
+          DEFAULT_BLOCK_GAS_LIMIT=7000000 STARKNET_NETWORK=http://0.0.0.0:1010 KAKAROT_ADDRESS=0x11c5faab8a76b3caff6e243b8d13059a7fb723a0ca12bbaadde95fb9e501bda deno test --allow-env --coverage=cov_profile_2 --allow-net indexer/src/transform.test.ts indexer/src/testnetData.test.ts
+          deno coverage cov_profile_2 --lcov > cov_profile_2.lcov
```

Comment:
If you use the json file "dump" I mention below, then I don't think we need to split the coverage computation in two runs.

Reply:
I split it to maintain compatibility with the old tests, which use KAKAROT_ADDRESS=0x1. But I can modify the other tests to use the correct address instead of 0x1. What do you think?

Reply:
Let's do that!

Reply:
Reverted the changes, now there is only one run with the correct KAKAROT_ADDRESS. The CI test fails only because the test modification cannot be executed in the main branch without merging the PR first